json-schemas/src/client-events-connections.json

```diff
@@ -77,6 +77,88 @@
       "type": "integer",
       "description": "The duration the connection was open for in milliseconds."
     },
+    "messages": {
```

Comment:
If we have these numbers on a per-attachment basis, then we can get the per-connection numbers by summing them, so is it redundant to also include these in connection events?

Reply:
Its easy to include both from the realtime side - so I guess its up to @ruimffl which is easier from the data side?

Reply:
If I understand correctly duration is already included in the connection events and we are using it to compute average and min/max duration when aggregating by day. It would be easier to keep it there instead of having to join to the attachment events. Hope I've answer the question (?)